src/crypto/public_key/elliptic/curves.js

```diff
@@ -251,21 +270,78 @@ async function webGenKeyPair(name) {
   const publicKey = await webCrypto.exportKey("jwk", webCryptoKey.publicKey);
 
   return {
-    pub: {
-      x: util.b64_to_Uint8Array(publicKey.x, true),
-      y: util.b64_to_Uint8Array(publicKey.y, true)
-    },
-    priv: util.b64_to_Uint8Array(privateKey.d, true)
+    publicKey: jwkToRawPublic(publicKey),
+    privateKey: util.b64_to_Uint8Array(privateKey.d, true)
   };
 }
 
 async function nodeGenKeyPair(name) {
   // Note: ECDSA and ECDH key generation is structurally equivalent
   const ecdh = nodeCrypto.createECDH(nodeCurves[name]);
   await ecdh.generateKeys();
-
   return {
-    pub: ecdh.getPublicKey().toJSON().data,
-    priv: ecdh.getPrivateKey().toJSON().data
+    publicKey: new Uint8Array(ecdh.getPublicKey()),
+    privateKey: new Uint8Array(ecdh.getPrivateKey())
+  };
+}
+
+const indutnyCurve = util.getFullBuild() ? function (name) {
+  return new indutnyEc.ec(name);
+} : undefined;
```

Comment:
Given that you already wrap the calling location of this function in `util.getFullBuild()` and how short it is, I think you could inline it.